engine/src/test/java/com/redhat/cloud/notifications/templates/common/TestComplianceTemplate.java

```diff
@@ -50,8 +50,9 @@ public void testInstantReportUploadFailedEmailBody() {
 
     @Test
     public void testInstantReportUploadFailedEmailTitle() {
+        eventTypeDisplayName = "Policy report failed to upload";
```

Comment:
This looks unused.

Reply:
Used by parent class [EmailTemplatesRendererHelper](https://github.com/RedHatInsights/notifications-backend/blob/master/engine/src/test/java/com/redhat/cloud/notifications/EmailTemplatesRendererHelper.java#L149).